src/lib/components/GitHubAvatar.svelte

```diff
@@ -5,7 +5,7 @@
 	export let size: number = 40;
 	export let asObjectUrl = false;
 
-	$: url = `https://avatars.githubusercontent.com/${githubAuthor}?v=${new Date().getTime()}`;
+	$: url = `https://avatars.githubusercontent.com/${githubAuthor}`;
```

Comment:
The query param acts as a cache buster, so we get the up to date image. Right now the cache headers would keep the image for 1 year